### DISCUSSION TITLE
markdown-it-abbr : shared markdown file

### DISCUSSION THREAD

**Discussion by akil-rails** [vuejs/vitepress#1172] [Q&A] [answered]
I am using the [markdown-it-abbr](https://github.com/markdown-it/markdown-it-abbr) .  This works by requiring the following kind of markdown
```
*[HTML]: Hyper Text Markup Language
*[W3C]:  World Wide Web Consortium
The HTML specification is maintained by the W3C.
```
When I have multiple pages that use the same abbreviation (HTML, W3C), I am having to repeat the definitions (HTML, W3C) across the pages.

Is there a way to import markdown files - so that I can consolidate all the abbreviations in a single shared .md file?

**Comment by brc-dd** [comment, marked as answer]
https://vitepress.vuejs.org/guide/markdown.html#markdown-file-inclusion
